core/package.json

```diff
@@ -7,7 +7,7 @@
 	},
 	"dependencies": {
 		"ajv": "^6.5.0",
-		"ajv-semver": "git+https://github.com/balena-io-modules/ajv-semver.git#resin-semver",
+		"ajv-semver": "^1.0.4",
```

Comment:
We might want to use this instead:
https://github.com/balena-io-modules/balena-semver

Reply:
@klutchell Looking at both of these modules, it appears that the ajv-semver module is not directly replaceable by balena-semver, if at all.